test/integration/index.spec.js

```diff
@@ -2246,72 +2315,6 @@ describe("Test CORS", () => {
 			.then(() => broker.stop()).catch(err => broker.stop().then(() => { throw err; }));
 	});
 
-	it("returns matching CORS origin wildcard with single origin", () => {
-		[broker, service, server] = setup({
```

Comment:
Why did you remove these test cases?

Reply:
I generally deleted few of the testcases because they overlap in testing with the new ones, but if you want i can restore them. I just thought that, since other test cases are here already, this would create noise in your test suite (as in: double testing a single feature/case + slowing down the suite)

Reply:
Ok, but please check that there are enough test cases to cover all if statements in the cors method. If yes, you can remove, if not, please add missing tests. Thanks